var/spack/repos/builtin/packages/py-mdanalysis/package.py

```diff
@@ -18,141 +18,52 @@ class PyMdanalysis(PythonPackage):
 
     maintainers("RMeli")
 
+    version("2.5.0", sha256="06ce4efab6ca1dbd2ee2959fc668049e1d574a8fe94ab948a4608244da1d016b")
     version("2.4.3", sha256="c4fbdc414e4fdda69052fff2a6e412180fe6fa90a42c24793beee04123648c92")
     version("2.4.2", sha256="ae2ee5627391e73f74eaa3c547af3ec6ab8b040d27dedffe3a7ece8e0cd27636")
-    version(
-        "1.0.0",
-        sha256="f45a024aca45e390ff1c45ca90beb2180b78881be377e2a1aa9cd6c109bcfa81",
-        deprecated=True,
-    )
-    version(
-        "0.20.1",
-        sha256="d04b71b193b9716d2597ffb9938b93f43487fa535da1bb5c1f2baccf356d7df9",
-        deprecated=True,
-    )
-    version(
-        "0.19.2",
-        sha256="c5395bbafa5efca2e1aee4715d26129844140c47cb8301da0293106cb969de7d",
-        deprecated=True,
-    )
-    version(
-        "0.19.1",
-        sha256="ff1d694f8598c0833ec340de6a6adb3b5e62b92d0fa94ee6401718ba972db3cc",
-        deprecated=True,
-    )
-    version(
-        "0.19.0",
-        sha256="248e3b37fc6150e31c609cc18a3927c32aee37b76d29cbfedf635e7e1aa982cf",
-        deprecated=True,
-    )
-    version(
-        "0.18.0",
-        sha256="a08acea1755112411e7db55e3f282e164b47a59e15794b38744cce6c596f252a",
-        deprecated=True,
-    )
-    version(
-        "0.17.0",
-        sha256="9bd61760334698cc7b8a57ad26456451e926e9c9e66722594ad8816561348cde",
-        deprecated=True,
-    )
-    version(
-        "0.16.2",
-        sha256="407d9a9ff1ab8a5e47973714d06fabff220f8d08a28792dee93e88e70e995b0a",
-        deprecated=True,
-    )
-    version(
-        "0.16.1",
-        sha256="3dc8f5d639ab3a0d152cbd7259ae9372ec8a9bac0f8cb7d3b80ce5adc1e3ee57",
-        deprecated=True,
-    )
-    version(
-        "0.16.0",
-        sha256="c4824fa1fddd336daa39371436187ebb023366885fb250c2827ed7fce2546bd4",
-        deprecated=True,
-    )
-    version(
-        "0.15.0",
-        sha256="9088786048b47339cba1f8a586977bbb3bb04ae1bcd0462b59e45bda37e25533",
-        deprecated=True,
-    )
 
     variant(
         "analysis",
         default=True,
         description="Enable analysis packages: matplotlib, scipy, seaborn",
     )
-    variant("amber", default=False, when="@:1.0.0", description="Support AMBER netcdf format.")
     variant("extra_formats", default=False, when="@2.4.0:", description="Support extra formats")
 
-    depends_on("python@2.7:", type=("build", "run"), when="@:1.0.0")
     depends_on("python@3.8:", type=("build", "run"), when="@2.4.0:")
+    depends_on("python@3.9:", type=("build", "run"), when="@2.5.0:")
 
     depends_on("py-setuptools", type="build")
 
-    depends_on("py-cython@0.16:", type="build", when="@:1.0.0")
     depends_on("py-cython@0.28:", type="build", when="@2.4.0:")
```

Comment:
```suggestion
    depends_on("py-cython@0.28:", type="build")
```
This can be done for almost all dependencies below too